raiden_contracts/data/source/services/MonitoringService.sol

```diff
@@ -342,11 +344,13 @@ contract MonitoringService is Utils {
         returns (address signature_address)
     {
         bytes32 message_hash = keccak256(abi.encodePacked(
-            "\x19Ethereum Signed Message:\n148",
+            "\x19Ethereum Signed Message:\n200",
+            monitoring_service_address,
```

Comment:
The reward is not bound to a specific monitoring service. Did you want to use monitoring_service_contract_address here?

Reply:
Ah, suddenly I remember that there should be a competition between MSs.

And yes, monitoring_service_contract_address is the right one.